### PR TITLE
fetch balances on each decision for the current account

### PR DESCRIPTION
**Pull request by mikec**
and store in state

### CODE REVIEW

**Comment by jpkcambridge** [comment]
👍

File: app/reducers/decisionBalances.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    case 'FETCH_TRADER_DECISION_BALANCES_SUCCESS':
+      return [
+        ..._.filter(
+          state,
+          balance => (
+            !(balance.decisionId == action.decisionId &&
+              balance.trader == action.trader)
+          )
+        ),
+        ...[{
```
Comment (emaG3m) [inline app/reducers/decisionBalances.js]: why do we need the `...[]` here? could you not just have `, { Object }`

(still getting used to es6!)

Comment (emaG3m) [reply app/reducers/decisionBalances.js]: not a big deal either way

**Comment by emaG3m** [comment]
👍

File: app/reducers/decisionBalances.js (hunk 1 above)
Comment (mikec) [reply app/reducers/decisionBalances.js]: @emaG3m if it's getting merged into another array it has to be an array... actually tried without it and it wouldn't let me!

**Pull request merged**